src/components/dialogs/staff-payment-error.vue

```diff
@@ -71,7 +71,7 @@ export default class StaffPaymentErrorDialog extends Vue {
 </script>
 
 <style lang="scss" scoped>
-@import "../../assets/scss/theme.scss";
+@import "../../assets/styles/theme.scss";
```

Comment:
This should be changed to the "@" format like the other files.

Reply:
By the way (for your information only), here is where the "@" shortcut is declared:
https://github.com/bcgov/business-create-ui/blob/25827cb57a6bae5089ff60dd79fb553cbf6e6a2f/tsconfig.json#L23